Library/Homebrew/fetch.rb

```diff
@@ -4,16 +4,23 @@
 module Homebrew
   # @api private
   module Fetch
-    sig { params(formula: Formula, args: CLI::Args).returns(T::Boolean) }
-    def fetch_bottle?(formula, args:)
+    sig {
+      params(
+        formula:                    Formula,
+        force_bottle:               T::Boolean,
+        bottle_tag:                 T.nilable(Symbol),
+        build_from_source_formulae: T::Array[String],
+      ).returns(T::Boolean)
+    }
+    def fetch_bottle?(formula, force_bottle:, bottle_tag:, build_from_source_formulae:)
       bottle = formula.bottle
 
-      return true if args.force_bottle? && bottle.present?
-      return true if args.bottle_tag.present? && formula.bottled?(args.bottle_tag&.to_sym)
+      return true if force_bottle && bottle.present?
+      return true if bottle_tag.present? && formula.bottled?(bottle_tag)
```

Comment:
```suggestion
      return true if bottle_tag && formula.bottled?(bottle_tag)
```

Reply:
`bottle_tag` can be an empty string so dropping `#present?` looks like it could change the behaviour of this method, which I'd rather not do.